.packit.yml

```
@@ -2,6 +2,8 @@ specfile_path: libmodulemd.spec
 upstream_package_name: libmodulemd
 downstream_package_name: libmodulemd
 upstream_tag_template: libmodulemd-{version}
+current_version_command:
+  - git describe --tags --match "*.*"  --abbrev=0 |perl -lne 'print "$1\n" if /^libmodulemd-(.*)/'
```

```suggestion
  - git describe --tags --match "*.*"  --abbrev=0 | sed -ne 's/^libmodulemd-//p'
```
I love `perl` as much as the next person, but `sed` always seems more appropriate in a shell script.

---

Huh... I was trying to make `sed` work here and it just wasn't coming together for me. I found the `perl` form on the web.

I agree, I'll use the `sed` form. (Better for dependencies too)